.github/workflows/python-ci-minimal.yml

```diff
@@ -0,0 +1,30 @@
+name: TileDB-SOMA Python CI (Minimal)
+
+on:
+  push:
```

Comment:
This now differs from r-ci.yml, which is not ideal.  And the GHA console shows push- and pull_request-triggered workflow runs somewhat differently.  I'm going to test once more with pull_request...